includes/batch.inc

```diff
@@ -674,18 +674,17 @@ function islandora_paged_content_get_missing_ocr_paged_content_objects() {
   // Base query.
   $query = <<<EOQ
 PREFIX islandora-rels-ext: <http://islandora.ca/ontology/relsext#>
-PREFIX fedora-model: <info:fedora/fedora-system:def/model#>
 PREFIX fedora-view: <info:fedora/fedora-system:def/view#>
 SELECT DISTINCT ?pid
 FROM <#ri>
 WHERE {
   ?page islandora-rels-ext:isPageOf ?pid .
-  ?pid fedora-model:label ?label .
+  ?pid fedora-view:lastModifiedDate ?date .
```

Comment:
Quick chat with Dan... This is only really done to ensure that `?pid` represents an existing object--according to the RI, at least.

This could be done more directly by checking that the base model is present... something like:
```
?pid fedora-model:hasModel <info:fedora/fedora-system:FedoraObject-3.0> .
```

Reply:
Yeah, forgot about the, er, ubiquity of lastModifiedDate ;) updated it so we 100% for realsies definitely get a Fedora object this time